### DISCUSSION TITLE
No aparece la descripción de la reserva

### DISCUSSION THREAD

**Discussion by Richar1481** [LibreBooking/librebooking#447] [General]
Necesito ayuda:

No aparece la descripción de mi reserva como invitado, solo aparece el color.

No se pueden ver los detalles de la reserva porque no tienes permisos para ninguno de los recursos en esta reserva

![Captura de pantalla_23-10-2024_94644_stjavila com mx](https://github.com/user-attachments/assets/36c90bd8-a1a1-4c6e-b2b9-94047730e340)

**Comment by treborin** [comment]
Hola @Richar1481 ,

Necesitaríamos más detales:

- Versión de la app Librebooking
- Versión de PHP
- Contenido de config.php

Tiene pinta de que es la configuración de las reservas, pero sin información no podremos decirte mucho.

**Comment by Richar1481** [comment]
Versión de la app Librebooking [GPLv3 v2.8.6.2]
Versión de PHP 8.2
$conf['settings']['app.title'] = 'Agenda ParroquialP';
$conf['settings']['default.timezone'] = 'Etc/UTC';
$conf['settings']['allow.self.registration'] = 'false';
$conf['settings']['admin.email'] = '';
$conf['settings']['admin.email.name'] = 'LB Administrator';
$conf['settings']['company.name'] = '';
$conf['settings']['company.url'] = '';
$conf['settings']['default.page.size'] = '50';
$conf['settings']['enable.email'] = 'true';
$conf['settings']['default.language'] = 'es';
$conf['settings']['script.url'] = '';
$conf['settings']['image.upload.directory'] = 'Web/uploads/images';
$conf['settings']['image.upload.url'] = 'uploads/images';
$conf['settings']['cache.templates'] = 'true';
$conf['settings']['use.local.js.libs'] = 'false';
$conf['settings']['registration.captcha.enabled'] = 'true';
$conf['settings']['registration.require.email.activation'] = 'false';
$conf['settings']['registration.auto.subscribe.email'] = 'false';
$conf['settings']['registration.notify.admin'] = 'false';
$conf['settings']['inactivity.timeout'] = '30';
$conf['settings']['name.format'] = '{first} {last}';
$conf['settings']['css.extension.file'] = '';
$conf['settings']['disable.password.reset'] = 'false';
$conf['settings']['home.url'] = '';
$conf['settings']['logout.url'] = '';
$conf['settings']['default.homepage'] = '1';
$conf['settings']['schedule']['use.per.user.colors'] = 'false';
$conf['settings']['schedule']['show.inaccessible.resources'] = 'true';
$conf['settings']['schedule']['reservation.label'] = '{name}';
$conf['settings']['schedule']['hide.blocked.periods'] = 'false';
$conf['settings']['schedule']['update.highlight.minutes'] = '0';
$conf['settings']['schedule']['show.week.numbers'] = 'false';
$conf['settings']['schedule']['fast.reservation.load'] = 'false';
$conf['settings']['ics']['subscription.key'] = '24c7a0ff6a9396696e6f';
$conf['settings']['ics']['future.days'] = '30';
$conf['settings']['ics']['past.days'] = '0';
$conf['settings']['privacy']['view.schedules'] = 'true';
$conf['settings']['privacy']['view.reservations'] = 'true';
$conf['settings']['privacy']['hide.user.details'] = 'true';
$conf['settings']['privacy']['hide.reservation.details'] = 'true';
$conf['settings']['privacy']['allow.guest.reservations'] = 'true';
$conf['settings']['reservation']['start.time.constraint'] = 'future';
$conf['settings']['reservation']['updates.require.approval'] = 'false';
$conf['settings']['reservation']['prevent.participation'] = 'false';
$conf['settings']['reservation']['prevent.recurrence'] = 'false';
$conf['settings']['reservation']['enable.reminders'] = 'false';
$conf['settings']['reservation']['allow.guest.participation'] = 'false';
$conf['settings']['reservation']['allow.wait.list'] = 'false';
$conf['settings']['reservation']['checkin.minutes.prior'] = '5';
$conf['settings']['reservation']['default.start.reminder'] = '';
$conf['settings']['reservation']['default.end.reminder'] = '';
$conf['settings']['reservation']['title.required'] = 'false';
$conf['settings']['reservation']['description.required'] = 'false';
$conf['settings']['reservation']['checkin.admin.only'] = 'false';
$conf['settings']['reservation']['checkout.admin.only'] = 'false';
$conf['settings']['reservation.notify']['resource.admin.add'] = 'false';
$conf['settings']['reservation.notify']['resource.admin.update'] = 'false';
$conf['settings']['reservation.notify']['resource.admin.delete'] = 'false';
$conf['settings']['reservation.notify']['resource.admin.approval'] = 'false';
$conf['settings']['reservation.notify']['application.admin.add'] = 'false';
$conf['settings']['reservation.notify']['application.admin.update'] = 'false';
$conf['settings']['reservation.notify']['application.admin.delete'] = 'false';
$conf['settings']['reservation.notify']['application.admin.approval'] = 'false';
$conf['settings']['reservation.notify']['group.admin.add'] = 'false';
$conf['settings']['reservation.notify']['group.admin.update'] = 'false';
$conf['settings']['reservation.notify']['group.admin.delete'] = 'false';
$conf['settings']['reservation.notify']['group.admin.approval'] = 'false';
$conf['settings']['uploads']['enable.reservation.attachments'] = 'false';
$conf['settings']['uploads']['reservation.attachment.path'] = 'uploads/reservation';
$conf['settings']['uploads']['reservation.attachment.extensions'] = 'txt,jpg,gif,png,doc,docx,pdf,xls,xlsx,ppt,pptx,csv';
$conf['settings']['database']['type'] = 'mysql';
$conf['settings']['database']['user'] = 'u328075763_1NNBh';
$conf['settings']['database']['password'] = '';
$conf['settings']['database']['hostspec'] = '127.0.0.1';
$conf['settings']['database']['name'] = 'u328075763_12Kx9';
$conf['settings']['phpmailer']['mailer'] = 'smtp';
$conf['settings']['phpmailer']['smtp.host'] = '';
$conf['settings']['phpmailer']['smtp.port'] = '25';
$conf['settings']['phpmailer']['smtp.secure'] = '';
$conf['settings']['phpmailer']['smtp.auth'] = 'true';
$conf['settings']['phpmailer']['smtp.username'] = '';
$conf['settings']['phpmailer']['smtp.password'] = '';
$conf['settings']['phpmailer']['sendmail.path'] = '/usr/sbin/sendmail';
$conf['settings']['phpmailer']['smtp.debug'] = 'false';
$conf['settings']['plugins']['Authentication'] = '';
$conf['settings']['plugins']['Authorization'] = '';
$conf['settings']['plugins']['Permission'] = '';
$conf['settings']['plugins']['PostRegistration'] = '';
$conf['settings']['plugins']['PreReservation'] = '';
$conf['settings']['plugins']['PostReservation'] = '';
$conf['settings']['install.password'] = '';
$conf['settings']['pages']['enable.configuration'] = 'true';
$conf['settings']['api']['enabled'] = 'false';
$conf['settings']['api']['allow.self.registration'] = 'false';
$conf['settings']['recaptcha']['enabled'] = 'false';
$conf['settings']['recaptcha']['public.key'] = '';
$conf['settings']['recaptcha']['private.key'] = '';
$conf['settings']['recaptcha']['request.method'] = 'curl';
$conf['settings']['email']['default.from.address'] = '';
$conf['settings']['email']['default.from.name'] = '';
$conf['settings']['reports']['allow.all.users'] = 'false';
$conf['settings']['password']['minimum.letters'] = '6';
$conf['settings']['password']['minimum.numbers'] = '0';
$conf['settings']['password']['upper.and.lower'] = 'false';
$conf['settings']['reservation.labels']['ics.summary'] = '{title}';
$conf['settings']['reservation.labels']['ics.my.summary'] = '{title}';
$conf['settings']['reservation.labels']['rss.description'] = '<div><span>Start</span> {startdate}</div><div><span>End</span> {enddate}</div><div><span>Organizer</span> {name}</div><div><span>Description</span> {description}</div>';
$conf['settings']['reservation.labels']['my.calendar'] = '{resourcename} {title}';
$conf['settings']['reservation.labels']['resource.calendar'] = '{name}';
$conf['settings']['reservation.labels']['reservation.popup'] = '';
$conf['settings']['security']['security.headers'] = 'false';
$conf['settings']['security']['security.strict-transport'] = 'true';
$conf['settings']['security']['security.x-frame'] = 'deny';
$conf['settings']['security']['security.x-xss'] = '1; mode=block';
$conf['settings']['security']['security.x-content-type'] = 'nosniff';
$conf['settings']['security']['security.content-security-policy'] = '';
$conf['settings']['google.analytics']['tracking.id'] = '';
$conf['settings']['authentication']['allow.facebook.login'] = 'false';
$conf['settings']['authentication']['allow.google.login'] = 'false';
$conf['settings']['authentication']['allow.microsoft.login'] = 'false';
$conf['settings']['authentication']['required.email.domains'] = '';
$conf['settings']['authentication']['hide.booked.login.prompt'] = 'false';
$conf['settings']['authentication']['captcha.on.login'] = 'false';
$conf['settings']['authentication']['google.client.id'] = '';
$conf['settings']['authentication']['google.client.secret'] = '';
$conf['settings']['authentication']['google.redirect.uri'] = '/Web/google-auth.php';
$conf['settings']['authentication']['microsoft.client.id'] = '';
$conf['settings']['authentication']['microsoft.tenant.id'] = 'common';
$conf['settings']['authentication']['microsoft.client.secret'] = '';
$conf['settings']['authentication']['microsoft.redirect.uri'] = '/Web/microsoft-auth.php';
$conf['settings']['authentication']['facebook.client.id'] = '';
$conf['settings']['authentication']['facebook.client.secret'] = '';
$conf['settings']['authentication']['facebook.redirect.uri'] = '/Web/facebook-auth.php';
$conf['settings']['credits']['enabled'] = 'false';
$conf['settings']['credits']['allow.purchase'] = 'false';
$conf['settings']['slack']['token'] = '';
$conf['settings']['tablet.view']['allow.guest.reservations'] = 'false';
$conf['settings']['tablet.view']['auto.suggest.emails'] = 'false';
$conf['settings']['registration']['require.phone'] = 'false';
$conf['settings']['registration']['require.position'] = 'false';
$conf['settings']['registration']['require.organization'] = 'false';
$conf['settings']['logging']['folder'] = 'uploads';
$conf['settings']['logging']['level'] = 'debug';
$conf['settings']['logging']['sql'] = 'false';
$conf['settings']['delete.old.data']['years.old.data'] = '3';
$conf['settings']['delete.old.data']['delete.old.announcements'] = 'false';
$conf['settings']['delete.old.data']['delete.old.blackouts'] = 'false';
$conf['settings']['delete.old.data']['delete.old.reservations'] = 'false';

**Comment by Richar1481** [comment]
![prueba2](https://github.com/user-attachments/assets/cef6e894-6c72-4cb1-b764-b4a266f65a17)

**Comment by treborin** [comment]
Hola @Richar1481 ,

Podrías cambiar esto:
`$conf['settings']['privacy']['hide.reservation.details'] = 'true';`

A esto (false):
`$conf['settings']['privacy']['hide.reservation.details'] = 'false';`

Elimina el contenido de la caché en la carpeta \tpl_c

**Reply by Richar1481** [reply]
Buen día.

Se realizo el procedimiento, sigue sin mostrar  los detalles. Pero se logro visualizar Recursos, pero no el **Título de la reserva** y **Descripción de la reserva**, no se pueden visualizar.

**Reply by Richar1481** [reply]
No se pueden ver los detalles de la reserva porque no tienes permisos para ninguno de los recursos en esta reserva

**Comment by tiagojulianoferreira** [comment]
We are facing the same challenge described in the thread, it seems to be related to recent updates, since in version 2.8 public reservations were displayed normally.

**Comment by vagrella** [comment]
We are using version 4.0. We are unable to configure the settings so that details are applied to unauthenticated users. Even when configured: 
privacy.view.schedules

privacy.hide.user.details

privacy.hide.reservation.details

How do we make them appear? Tks!
